data/.gitignore

```diff
@@ -0,0 +1,3 @@
+/train.tsv
```

Comment:
Okay, so this way we don’t ignore everything that’s in here and I also think it’s better to have a single gitignore for the project, so my suggestion is this:

- yeet this file
- add an empty `.gitkeep` file in here

add the following to the general `.gitignore`:

```
# Data folder
 data/*
 !data/.gitkeep
```

Reply:
nice, thanks @sarajuhosova! I did this and I believe it also resolves @wouterpolet's comment above :)